packages/framework/src/Component/Elasticsearch/Exception/ElasticsearchIndexException.php

```diff
@@ -0,0 +1,154 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Component\Elasticsearch\Exception;
+
+use Exception;
+
+class ElasticsearchIndexException extends Exception
+{
+    /**
+     * @param string $documentName
+     * @param string $definitionFilename
+     * @return self
+     */
+    public static function invalidJsonInDefinitionFile(string $documentName, string $definitionFilename): self
+    {
+        return new self(sprintf(
+            'Invalid JSON in %s definition file %s',
+            $documentName,
+            $definitionFilename
+        ));
+    }
+
+    /**
+     * @param string $definitionFilename
+     * @return self
+     */
+    public static function cantReadDefinitionFile(string $definitionFilename): self
+    {
+        return new self(sprintf(
+            'Can\'t read definition file at path %s. Please check that file exists and has permissions for reading.',
+            $definitionFilename
+        ));
+    }
+
+    /**
+     * @param string $indexName
+     * @param array $error
+     * @return self
+     */
+    public static function createIndexError(string $indexName, array $error): self
+    {
+        return new self(sprintf(
+            'Error when creating index %s:' . PHP_EOL . '%s',
```

Comment:
I suggest putting index name into quotation marks in the whole file, so it's easier to spot spaces or any other unwanted characters.

(like `'Error when creating index "%s":' . PHP_EOL . '%s',`)

Reply:
Nice catch. Totally agree.